src/components/tree/infoPanels/hover.js

```diff
@@ -110,31 +110,36 @@ const getMutationsJSX = (d, mutType) => {
       const nDisplay = 9; // max number of mutations to display
       const nGapDisp = 4; // max number of gaps/Ns to display
 
-      //gather muts with N/-
-      let ngaps = d.muts.filter(mut => {return mut.slice(-1) == "N" || mut.slice(-1) == "-"
-                                        || mut.slice(0) == "N" || mut.slice(0) == "-";})
+      // gather muts with N/-
+      const ngaps = d.muts.filter((mut) => {
+        return mut.slice(-1) === "N" || mut.slice(-1) === "-"
```

Comment:
String comparisons should always be done with `===` or `!==`: https://stackoverflow.com/questions/3586775/what-is-the-correct-way-to-check-for-string-equality-in-javascript